drivers/ethernet/eth_stm32_hal_priv.h

```diff
@@ -141,20 +136,24 @@ struct eth_stm32_hal_dev_data {
 #endif
 };
 
-void setup_mac_filter(ETH_HandleTypeDef *heth);
-int eth_tx(const struct device *dev, struct net_pkt *pkt);
-struct net_pkt *eth_rx(const struct device *dev);
-int eth_hal_init(const struct device *dev);
-void set_mac_config(const struct device *dev, struct phy_link_state *state);
+void eth_stm32_setup_mac_filter(ETH_HandleTypeDef *heth);
+void eth_stm32_set_mac_config(const struct device *dev, struct phy_link_state *state);
+int eth_stm32_eth_tx(const struct device *dev, struct net_pkt *pkt);
+struct net_pkt *eth_stm32_rx(const struct device *dev);
+int eth_stm32_hal_init(const struct device *dev);
 int eth_stm32_hal_start(const struct device *dev);
 int eth_stm32_hal_stop(const struct device *dev);
 int eth_stm32_hal_set_config(const struct device *dev,
 			     enum ethernet_config_type type,
 			     const struct ethernet_config *config);
-struct net_if *get_iface(struct eth_stm32_hal_dev_data *ctx);
+struct net_if *eth_stm32_get_iface(struct eth_stm32_hal_dev_data *ctx);
 #if defined(CONFIG_ETH_STM32_MULTICAST_FILTER)
 void eth_stm32_mcast_filter(const struct device *dev,
 			    const struct ethernet_filter *filter);
 #endif /* CONFIG_ETH_STM32_MULTICAST_FILTER */
+#ifdef CONFIG_PTP_CLOCK_STM32_HAL
```

Comment:
an empty line above would be welcome IMHO. Ditto at line 150.

Reply:
Done